code/game/atoms_movable.dm

```diff
@@ -9,7 +9,7 @@
 	var/last_move = null
 	var/anchored = 0
 	// var/elevation = 2    - not used anywhere
-	var/move_speed = 10
+	var/move_speed = 1
```

Comment:
This var was previously not speed at all, but the time since last move.
I've fixed it to actually be what it should be, and in the process fixed the exactly one place it was used in code